pkg/kn/commands/service/list_mock_test.go

```diff
@@ -17,60 +17,21 @@ package service
 import (
 	"strings"
 	"testing"
-	"time"
 
 	"gotest.tools/assert"
-	"k8s.io/apimachinery/pkg/api/errors"
+	"knative.dev/serving/pkg/apis/serving/v1alpha1"
+
 	knclient "knative.dev/client/pkg/serving/v1alpha1"
 	"knative.dev/client/pkg/util"
-	"knative.dev/client/pkg/wait"
-
-	"knative.dev/serving/pkg/apis/serving/v1alpha1"
+	"knative.dev/client/pkg/util/mock"
 )
 
 func TestServiceListAllNamespaceMock(t *testing.T) {
-	client := knclient.NewMockKnClient(t)
-
+	client := knclient.NewMockKnServiceClient(t, "default")
 	r := client.Recorder()
+	setupListExpectations(r)
 
-	r.GetService("svc1", nil, errors.NewNotFound(v1alpha1.Resource("service"), "svc1"))
```

Comment:
removed that parts as they are not about the list test and already covered in the create tests.